database.php

```diff
@@ -0,0 +1,7 @@
+<?php
+$connection = mysqli_connect("localhost", "root", "", "books_db");
```

Comment:
Observations: Using MySQLi procedural style is outdated.
Recommendations: Consider using PDO or MySQLi object-oriented style for better maintainability and flexibility.

```php
$connection = new mysqli("localhost", "root", "", "books_db");
if ($connection->connect_error) {
    die("Failed to connect to database: " . $connection->connect_error);
}
```